### ISSUE TITLE
Version mismatch

### ISSUE THREAD

**Issue by piotrd** [danialfarid/ng-file-upload#1170]
The following, although file name says `10.1.0.tar.gz`, actually downloads and installs ng-file-upload version **10.0.4**:

```
bower install ng-file-upload -f
bower ng-file-upload#*         resolve git://github.com/danialfarid/angular-file-upload-bower.git#*
bower ng-file-upload#*        download https://github.com/danialfarid/angular-file-upload-bower/archive/10.1.0.tar.gz
```

@danialfarid, is that some Github cache issue or an incorrect version was built into these archives?

**Comment by danialfarid** [comment]
It seems fine on the githuhb: https://github.com/danialfarid/ng-file-upload-bower/releases

**Comment by danialfarid** [comment]
Fixed at 10.1.2
